packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/technical-details/technical-details.html

```diff
@@ -13,15 +13,23 @@
         term="{{:: 'dedicated_server_dashboard_technical_details_ram' | translate }}"
     >
         <oui-tile-description>
-            <span data-ng-bind=":: $ctrl.formattedRAM"></span>
+            <p data-ng-bind=":: $ctrl.formattedRAM"></p>
+        </oui-tile-description>
+        <oui-tile-description>
+            <span
+                data-ng-if="$ctrl.upgradeTask.isUpgradeScheduled('ram')"
+                class="oui-badge oui-badge_warning m-0"
+                data-translate="dedicated_server_dashboard_technical_details_upgrade_in_progress"
+                data-translate-values="{ schedule: $ctrl.upgradeTask.scheduledTimeSlot }"
+            ></span>
         </oui-tile-description>
     </oui-tile-definition>
     <oui-tile-button
-        ng-if="$ctrl.isRamUpgradable()"
-        on-click="$ctrl.goToManualUpgrade('ram')"
+        ng-if="$ctrl.isRamUpgradable() || $ctrl.upgradeTask.shouldScheduleUpgrade('ram')"
```

Comment:
move 'ram' to constants. Its been used in multiple places

Reply:
Done